src/components/tx-flow/flows/SignMessage/SignMessage.tsx

```diff
@@ -220,13 +238,15 @@ const SignMessage = ({ message, safeAppId, requestId }: ProposeProps | ConfirmPr
           </Typography>
           <DecodedMsg message={decodedMessage} isInModal />
 
-          <Accordion sx={{ mt: 2, '&.Mui-expanded': { mt: 2 } }}>
+          <Accordion sx={{ mb: 2, mt: 2, '&.Mui-expanded': { mt: 2 } }}>
             <AccordionSummary expandIcon={<ExpandMoreIcon />}>SafeMessage details</AccordionSummary>
             <AccordionDetails>
               <MessageHashField label="SafeMessage" hashValue={safeMessageMessage} />
               <MessageHashField label="SafeMessage hash" hashValue={safeMessageHash} />
             </AccordionDetails>
           </Accordion>
+
+          <Redefine />
```

Comment:
The info icon says on hover `This transaction has been automatically scanned for risks to help prevent scams. ...`. In this case `message` instead of `transaction` would be more fitting. Wdyt?